app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3:
+        raise ValueError("Invalid command")
+
+    if parts[0] != "mv":
```

Comment:
```suggestion
    cmd, source_path, destination_path = parts
    if cmd != "mv":
```
And the use this variables

Reply:
Unzipiing better than [0], [1]....

Reply:
agree with you